pkg/backup/backup.go

```diff
@@ -244,6 +249,15 @@ func (kb *kubernetesBackupper) BackupWithResolvers(
 	log.Infof("Including namespaces: %s", backupRequest.NamespaceIncludesExcludes.IncludesString())
 	log.Infof("Excluding namespaces: %s", backupRequest.NamespaceIncludesExcludes.ExcludesString())
 
+	// check if there are any namespaces included in the backup which are managed by argoCD
+	// We will check for the existence of a label "argocd.argoproj.io/managed-by" in the includedNamespaces and add a warning
```

Comment:
would remove label value here

If one day it changes, we need to change it here and on line 77

Reply:
Alternatively import from https://github.com/argoproj/argo-cd/blob/1dcf3a56653c907f8b6d6dbadc8e2c66a61f3236/common/common.go#L186-L188

Reply:
nvm.. +1

Reply:
Done, updated